sidebars.js

```diff
@@ -1,43 +1,14 @@
 module.exports = {
   Guides: [
     "guides/introduction-to-camunda",
+    "guides/getting-started-example",
+    "guides/getting-started-java-spring",
```

Comment:
Given that we have the new getting started, which also uses spring (https://preview.docs.camunda.cloud/pr-6073/docs/next/guides/getting-started-example/), can we not simply delete the old one (https://preview.docs.camunda.cloud/pr-6073/docs/next/guides/getting-started-java-spring/).

@megglos , wdyt?